packages/eds-core-react/src/components/Datepicker/calendars/CalendarHeader.tsx

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components'
+import { CalendarState, RangeCalendarState } from '@react-stately/calendar'
+import { Button, Icon } from '../../../index'
+import {
+  chevron_down,
+  chevron_left,
+  chevron_right,
+  chevron_up,
+} from '@equinor/eds-icons'
+import { CalendarDate } from '@internationalized/date'
+import { tokens } from '@equinor/eds-tokens'
+
+const HeaderWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: capitalize;
+  width: 100%;
+`
+
+function TodayPicker({
+  onClick,
+  disabled,
+}: {
+  onClick: (v: CalendarDate) => void
+  disabled: boolean
+}) {
+  const today = new Date()
+  return (
+    <Button
+      disabled={disabled}
+      onClick={() =>
+        onClick(
+          new CalendarDate(
+            today.getFullYear(),
+            today.getMonth() + 1,
+            today.getDate(),
+          ),
+        )
+      }
+      variant={'ghost'}
+      style={{ marginLeft: 16 }}
+    >
+      Today
+    </Button>
+  )
+}
+
+/**
+ * The default header for the calendar components if no custom header is provided
+ */
+export function CalendarHeader({
+  state,
+  title,
+  previousMonthDisabled,
+  nextMonthDisabled,
+  showYearPicker,
+  setShowYearPicker,
+}: {
+  state: CalendarState | RangeCalendarState
+  title: string
+  previousMonthDisabled?: boolean
+  nextMonthDisabled?: boolean
+  showYearPicker: boolean
+  setShowYearPicker: (showYearPicker: boolean) => void
+}) {
+  return (
+    <HeaderWrapper>
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          width: '100%',
+        }}
```

Comment:
Should this be moved to a styled-component?

Reply:
Possibly; I was a bit lazy when I added more styles here